src/main/java/uk/gov/hmcts/darts/common/entity/ObjectAdminActionEntity.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "object_admin_action")
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class ObjectAdminActionEntity {
+
+    @Id
+    @Column(name = "oaa_id")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "oaa_gen")
+    @SequenceGenerator(name = "oaa_gen", sequenceName = "oaa_seq", allocationSize = 1)
+    private Integer id;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = false)
```

Comment:
just checking - do these definitely need to be EAGER?

Reply:
done